cypress.json

```diff
@@ -2,7 +2,8 @@
   "baseUrl": "https://cypress.ci.folio.org",
   "viewportWidth": 1920,
   "viewportHeight": 1080,
-  "video": false,
+  "video": true,
+  "videosFolder": "cypress/videos",
```

Comment:
Do we need to record a video in our cypress environment while running the tests?

Reply:
yeap, we can record a video and attach it to Jira ticket if PO will want see video